usagov/usagov-2021
# USAGOV-2038: disable page cache for tome

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2038

## Description

Disables the page and menu caches when a tome command runs.

Same as #2040, but put the new file in a directory that actually gets copied into the docker image. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

Tome output should be unaffected. Compare exports between this and a run from dev. 

Validate deployment to dev: Tome should run without error (when the nonlocal.services.yml file didn't make it into the image, tome immediately failed with a message about the NULL cache backend not being defined). Screenshot of success: 
![Screenshot 2024-11-12 at 6 18 07 PM](https://github.com/user-attachments/assets/6ee0471a-d36a-4148-aec5-e2faa5fc5a14)

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
